tests/test_hvac_csc.py

```diff
@@ -0,0 +1,275 @@
+# This file is part of ts_hvac.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import asynctest
+import logging
+
+from lsst.ts import salobj
+from lsst.ts import hvac
+
+STD_TIMEOUT = 2  # standard command timeout (sec)
+
+logging.basicConfig(format="%(asctime)s:%(levelname)s:%(name)s:%(message)s", level=logging.DEBUG)
+
+
+class CscTestCase(salobj.BaseCscTestCase, asynctest.TestCase):
+    def basic_make_csc(self, initial_state, config_dir, simulation_mode, **kwargs):
+        return hvac.HvacCsc(
+            initial_state=initial_state, config_dir=config_dir, simulation_mode=simulation_mode,
+        )
+
+    async def test_standard_state_transitions(self):
+        async with self.make_csc(initial_state=salobj.State.STANDBY, config_dir=None, simulation_mode=1):
+            await self.check_standard_state_transitions(
+                enabled_commands=(
+                    "chiller01P01",
+                    "crack01P02",
+                    "fancoil01P02",
+                    "manejadoraLower01P05",
+                    "manejadoraSblancaP04",
+                    "vea01P01",
+                    "vea01P05",
+                    "vea08P05",
+                    "vea09P05",
+                    "vea10P05",
+                    "vea11P05",
+                    "vea12P05",
+                    "vea13P05",
+                    "vea14P05",
+                    "vea15P05",
+                    "vea16P05",
+                    "vea17P05",
+                    "vec01P01",
+                    "vex03P04",
+                    "vex04P04",
+                    "vin01P01",
+                ),
+            )
```

Comment:
If you do end up with a list of these in the CSC then you could use that list here. Though then the best way to support special cases is probably to use the full list and skip exclusions, if any. Not a bad idea in any case.